articles/virtual-desktop/apply-windows-license.md

```diff
@@ -55,3 +55,7 @@ Run the following cmdlet to see a list of all session host VMs that have the Win
 $vms = Get-AzVM
 $vms | Where-Object {$_.LicenseType -like "Windows_Client"} | Select-Object ResourceGroupName, Name, LicenseType
 ```
+
```

Comment:
Remove the extra line break after the code block, then add a line break between the H2 ("## Requirements...) and the body paragraph ("If you deploy Windows Server....").